hack/go-tools/verify-gofmt.sh

```diff
@@ -27,7 +27,7 @@ find_files() {
     \) -name '*.go'
 }
 
-GOFMT="gofmt -s"
+GOFMT="gofmt -s -w"
```

Comment:
I'm not sure this is quite what we want to do: this script is run during a Prow job, and having it do the writes in place does not help us there. Does `gofmt -s -w` also write out the names of the files that it simplifies?

I think, to add this line, it would make sense to add an `update-gofmt.sh` script that involved the `gofmt` command correctly.

Reply:
@perotinus I only updated the promoted message in the new pr, can you help check? The major improvement that I want to do is to give developer a good message for the next action.

Reply:
@gyliu513 Sure, this looks good to me. Thank yoU!

Can you merge the two commits so that there isn't noise in the change?

Reply:
seems squash does not work, created another PR at https://github.com/kubernetes/cluster-registry/pull/251